frontend/src/actionCreators/parcelsActionCreator.ts

```diff
@@ -167,17 +167,23 @@ const parcelDeletingToasts: LifecycleToasts = {
   errorToast: pimsToasts.parcel.PARCEL_DELETING_ERROR,
 };
 
-export const deleteParcel = (parcel: IParcel) => (dispatch: Function) => {
+/**
+ * Make an AJAX request to delete the specified 'parcel' from inventory.
+ * @param parcel IParcel object to delete from inventory.
+ */
+export const deleteParcel = (parcel: IParcel) => async (dispatch: Function) => {
```

Comment:
Converted to async/await.  For some reason our `customAxios` doesn't return an error object.  It's always `undefined` in this `try/catch`...

Reply:
Custom axios may be doing some error handling itself? or you need to do something different when doing an await ie pass a parameter to axios or something.